src/components/VmDetails/BaseCard.js

```diff
@@ -90,7 +113,7 @@ class BaseCard extends React.Component {
             <CardEditButton tooltip={editTooltip} editEnabled={editing} onClick={this.clickEdit} />
           )}
 
-          {children({ isEditing: editing })}
+          <this.renderChildren isEditing={editing} />
```

Comment:
This is not right calling react component. Please, assign it to capitalize variable before use.
https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized

Reply:
Makes sense.  I'll make that change.